cmd/mesh/manifest-apply.go

```diff
@@ -38,7 +38,7 @@ type manifestApplyArgs struct {
 	readinessTimeout time.Duration
 	// wait is flag that indicates whether to wait resources ready before exiting.
 	wait bool
-	// yes means don't ask for confirmation (asking for confirmation not implemented)
+	// yes means don't ask for confirmation when manifest apply
```

Comment:
"yes means don't..." is a little confusing. What about "If true, apply the manifest without asking for confirmation. If false, apply will ask for an explicit confirmation before modifying the cluster state."?